src/app/lib/config/services/arabic.js

```diff
@@ -16,7 +16,7 @@ const service = {
     articleTimestampPrefix: 'جدّد في',
     atiAnalyticsAppName: 'news-arabic',
     atiAnalyticsProducerId: '5',
-    brandName: 'BBC News عربي',
+    brandName: 'BBC News',
     serviceLocalizedName: 'عربي',
```

Comment:
Could you place the `product` property above this, so we keep it consistent across files? :)

Reply:
good spot, fixing now 😄

Reply:
Yeah, it already existed. I meant to move it down between those two 😄